app/services/notifications/setup/submit-check.service.js

```diff
@@ -0,0 +1,40 @@
+'use strict'
+
+/**
+ * Orchestrates handling the data for `/notifications/setup/{sessionId}/check` page
+ * @module SubmitCheckService
+ */
+
+const SessionModel = require('../../../models/session.model.js')
+const RecipientsService = require('./fetch-recipients.service.js')
+const DetermineRecipientsService = require('./determine-recipients.service.js')
+const { currentTimeInNanoseconds, calculateAndLogTimeTaken } = require('../../../lib/general.lib.js')
+
+/**
+ * Orchestrates handling the data for `/notifications/setup/{sessionId}/check` page
+ *
+ * This service will transform the recipients into notifications and start processing notifications.
+ *
+ * @param {string} sessionId - The UUID for the notification setup session record
+ *
+ * Does not return to allow the process to run in the background
```

Comment:
If you want to keep this note, it should be bumped above the params within the description.

But I think there are enough examples of this pattern now that it is also fine to drop it 🤷